Assets/_Functionalities/Wms/Scripts/ATMMapLayerGameObject.cs

```diff
@@ -0,0 +1,30 @@
+namespace Netherlands3D.Twin.Layers
+{
+    /// <summary>
+    /// Extention of LayerGameObject that injects a 'streaming' dataprovider WMSTileDataLayer
+    /// </summary>
+    public class ATMMapLayerGameObject : CartesianTileLayerGameObject
+    {
+
+        protected override void Start()
+        {
+            base.Start();
+
+            LayerData.LayerOrderChanged.AddListener(SetRenderOrder);
+        }
+
```

Comment:
Er bestaat al een ATMLayerGameObject?

Reply:
Klopt, dit is de parent. In de originele code was er een CartesianTileLayer genest in een CartesianTileLayer omdat het dezelfde monobehaviour was - deze is toen gesplitst in een AtmMapLayerGameObject (dat geen CartesianTileLayer is) en een AtmLayerGameObject dat wel een cartesian tile layer game object is; daardoor heb je geen nesting en geen onverwacht gedrag van de functionaliteit van de CartesianTileLayer (ik dacht dat dit o.a. bijdroeg aan het probleem met de positionering)